src/Sarif.Viewer.VisualStudio/String.extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Sarif.Viewer
+{
+    static class StringExtensions
+    {
+        public static KeyValuePair<string, string> KeyWithValue(this string key, string value)
```

Comment:
>KeyValuePair [](start = 22, length = 12)

where are the unit tests for this admittedly simple method?

pretty easy to test. :)